src/utils/websockets.js

```diff
@@ -1,5 +1,9 @@
-import SockJs from 'sockjs-client';
-import Stomp from 'stompjs/lib/stomp.min.js';
+import SockJS from 'sockjs-client';
+import { Client } from '@stomp/stompjs';
+
+function socketFactory() {
+  return new SockJS(config.WS_URL);
```

Comment:
Where is the config defined ? Here this is undefined

Reply:
config is a global in `config.js`